### PR TITLE
Jump nav functionality

### PR DESCRIPTION
**Pull request by nicklee**

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/molecules/jump-nav/jump-nav.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div class="jump-nav tab-nav {{ modifier }} {% if sticky %}sticky{% endif %}">
+    <div class="jump-nav__layout tab-nav__layout grid">
+        <div class="jump-nav__heading">
+            <div class="jump-nav__heading-label">
+                <span class="jump-nav__pre-stuck-heading">Jump to</span>
+                <span class="jump-nav__stuck-heading">{{ title }}</span>
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/molecules/jump-nav/jump-nav.html:6]: guessing there will always be a title so this doesn't need a conditional around it?

File: rca/static_src/sass/components/_tab-item.scss [hunk 2]
```diff
@@ -58,4 +65,21 @@
...
+
+    .js-is-sticky.jump-nav & {
+        color: $color--meta-lightbg;
+
+        &:hover {
+            color: $color--black;
+        }
+
+        &.is-active {
+            color: $color--black;
```
Comment (chris-lawton) [inline rca/static_src/sass/components/_tab-item.scss:77]: meganitpick: could combine these up to save a few lines 

```
        &:hover,
        &.is-active {
            color: $color--black;
        }
```

**Review by chris-lawton — APPROVED** [review]

File: rca/project_styleguide/templates/patterns/molecules/jump-nav/jump-nav.html (hunk 1 above)
Comment (nicklee) [reply rca/project_styleguide/templates/patterns/molecules/jump-nav/jump-nav.html:6]: I'll add one in, won't do any harm :) Cheers

File: rca/static_src/sass/components/_tab-item.scss (hunk 2 above)
Comment (nicklee) [reply rca/static_src/sass/components/_tab-item.scss:71]: 👍

**Pull request closed without merging**